error-monitoring/static/error-list.html

```diff
@@ -172,12 +196,37 @@ <h2>
 
       <pre class="stacktrace"><code>{{& stacktrace}}</code></pre>
 
-      <a target="_blank"
-        rel="noopener noreferrer"
-        href="{{createUrl}}"
-        onclick="this.parentNode.classList.add('error-report-done')">
-        <button class="button-blue button-create">Create issue</button>
-      </a>
+      <div class="error-actions">
+        <a target="_blank"
+          rel="noopener noreferrer"
+          href="{{createUrl}}"
+          onclick="this.parentNode.classList.add('error-report-done')">
+          <button class="button-blue button-create">
+            Create new issue
+          </button>
+        </a><!--
+
+     --><form method="GET"
```

Comment:
<!--?

Reply:
<!-- whitespace here would put a weird gap between the buttons/throw off the sizing -->